libafl/src/feedbacks/map.rs

```diff
@@ -472,10 +485,12 @@ where
                 .enumerate()
                 .filter(|(_, value)| *value != initial)
             {
+                if history_map[i] == initial {
```

Comment:
should we increment this after updating history_map[i]?

Reply:
I don't think so, because the reduce on the next line would set history_map to a non-initial value, right?